.travis.yml

```diff
@@ -33,3 +33,4 @@ before_install:
 
 script: |
     travis_wait cargo build --verbose --features "build"
```

Comment:
I don't think we need both.

Reply:
I guess `cargo build --examples...` should be enough? :)